badge_server/test_badges.py

```diff
@@ -82,6 +82,13 @@
     status=compatibility_store.Status.SUCCESS,
     timestamp=datetime.datetime.utcnow())
 
+APACHE_BEAM_GOOGLE_API_CORE_RECENT_SUCCESS_3 = compatibility_store.CompatibilityResult(
```

Comment:
The name has "SUCCESS" in it.

Reply:
Do you need all of the constants below? For the tests that you added, don't need you just need two constants?

Reply:
I thought it would be good to get all of the permutations, but if you don't think that's necessary, I can remove some.